Project.toml

```
@@ -13,6 +13,7 @@ UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 
 [compat]
 AWSCore = "0.6"
+MbedTLS = "0.7"
```

```suggestion
MbedTLS = "0.5, 0.6, 0.7"
```
We are good with every tagged version of MbedTLS, since we only use MbedException which has been around forever.

---

Thanks. I've gotten into the habit of dropping older revisions which didn't have a Project.toml.